README.md

```diff
@@ -2,9 +2,15 @@
 
 This is a feature plugin for a modern, javascript-driven WooCommerce Admin experience.
 
+---
+
+:warning: This project is in active development, and is not ready for general use. You can follow the features in development by looking at the [project's issues](https://github.com/woocommerce/wc-admin/issues). **We do not recommend running this on production sites.**
+
+---
+
 ## Prerequisites
 
-[Gutenberg](https://wordpress.org/plugins/gutenberg/) and [WooCommerce](https://wordpress.org/plugins/woocommerce/) should be installed prior to activating the WooCommerce Admin feature plugin. You also need v3 of the WooCommerce REST API via [wc-api-dev](https://github.com/woocommerce/wc-api-dev/) or [the feature branch](https://github.com/woocommerce/woocommerce/tree/feature/rest-api-v3).
+[Gutenberg](https://wordpress.org/plugins/gutenberg/) and [WooCommerce](https://wordpress.org/plugins/woocommerce/) should be installed prior to activating the WooCommerce Admin feature plugin. You also need v3 of the WooCommerce REST API via [wc-api-dev](https://github.com/woocommerce/wc-api-dev/) or [the feature branch of WooCommerce](https://github.com/woocommerce/woocommerce/tree/feature/rest-api-v3).
```

Comment:
I think we should just remove the `wc-api-dev` option and point people at the feature branch, otherwise this is looking great. Thanks @ryelle

Reply:
I was wondering about that - i'll remove it.